src/charsmap_normalization.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "charsmap_normalization.hpp"
+#include "utils.hpp"
+#include "sentencepiece_trainer.h"  // for making normalizer spec
+#include "absl/strings/str_format.h"
+
+using namespace ov;
+
+namespace {
+    sentencepiece::NormalizerSpec make_identity_spec() {
+        return sentencepiece::SentencePieceTrainer::GetNormalizerSpec("identity");
```

Comment:
please, link `sentencepiece_train-static` to use train symbols (see https://github.com/google/sentencepiece/blob/d8f741853847553169444afc12c00f4bbff3e9ce/src/CMakeLists.txt#L226)

Reply:
Fixed in https://github.com/openvinotoolkit/openvino_tokenizers/pull/272